tests/attr/helpers/basic_models.py

```diff
@@ -212,9 +213,9 @@ def __init__(
         self.relu = nn.ReLU()
         self.linear2 = nn.Linear(hidden_dim, output_dim)
 
-    def forward(self, input, another_input=None):
-        embedding1 = self.embedding1(input)
```

Comment:
I was using the same inputs for different embeddings. Changing it to use different inputs for each embedding.